### PR TITLE
Prod-release

### PR DESCRIPTION
**Pull request by rosvik**

### CODE REVIEW

**Comment by mikaelbr** [comment]
There are some new configuration variables. Are these added in the infra-setup, @torfjor ?

**Comment by rosvik** [comment]
Also the NFK build is failing, is that an issue @mikaelbr?

File: cloudbuild.yaml [hunk 1]
```diff
@@ -15,12 +15,16 @@ steps:
...
       - europe-west1
-      - '--set-env-vars=ENTUR_BASEURL=$_ENTUR_BASEURL,PERIOD_TICKET_INVITE_KEY=$_PERIOD_TICKET_INVITE_KEY,TICKET_INVITE_KEY=$_TICKET_INVITE_KEY'
+      - '--set-env-vars=ENTUR_BASEURL=$_ENTUR_BASEURL,PERIOD_TICKET_INVITE_KEY=$_PERIOD_TICKET_INVITE_KEY,TICKET_INVITE_KEY=$_TICKET_INVITE_KEY,FLEX_TICKET_INVITE_KEY=$_FLEX_TICKET_INVITE_KEY,VIPPS_BASE_URL=$_VIPPS_BASE_URL,VIPPS_CALLBACK_URL=$_VIPPS_CALLBACK_URL'
+      - --set-secrets=VIPPS_CLIENT_ID=bff-vipps-client-id:latest,VIPPS_CLIENT_SECRET=bff-vipps-client-secret:latest
       - --platform
       - managed
       - --allow-unauthenticated
 substitutions:
+  _VIPPS_CALLBACK_URL: 'atb://auth/vipps'
+  _VIPPS_BASE_URL: 'https://apitest.vipps.no'
```
Comment (torfjor) [inline cloudbuild.yaml:25]: This must be changed to the Vipps production environment.

**Comment by torfjor** [comment]
Secrets exist in the production project.

Added an override for substitution `_VIPPS_API_BASE_URL`.

🚀

**Comment by rosvik** [comment]
Waiting for test in staging from @tormoseng before we merge this

**Pull request merged**